### ISSUE TITLE
Elasticsearch 7.x support

### ISSUE THREAD

**Issue by CorrosiveKid** [barseghyanartur/django-elasticsearch-dsl-drf#148]
Elasticsearch is now at 7.3 and it would be great to have support for Elasticsearch 7.x clusters. The underlying [django-elasticsearch-dsl](https://github.com/sabricot/django-elasticsearch-dsl) package supports Elasticsearch 7.x too as of [11th August 2019](https://github.com/sabricot/django-elasticsearch-dsl/releases/tag/7.0.0).

**Comment by barseghyanartur** [comment]
WIP in [es7](https://github.com/barseghyanartur/django-elasticsearch-dsl-drf/tree/es7) branch.

**Comment by CorrosiveKid** [comment]
Only 1 test failed for me with Python 3.7 and Django 2.2. I'll see what I can do to help.
```
tests/test_faceted_search.py .                                                               [  0%]
tests/test_filtering_common.py ..........................ss.                                 [ 19%]
tests/test_filtering_geo_spatial.py .........                                                [ 24%]
tests/test_filtering_global_aggregations.py .                                                [ 25%]
tests/test_filtering_nested.py ..............ss                                              [ 35%]
tests/test_filtering_post_filter.py ........................ss.                              [ 52%]
tests/test_functional_suggesters.py ..                                                       [ 54%]
tests/test_helpers.py F                                                                      [ 54%]
tests/test_highlight.py .                                                                    [ 55%]
tests/test_more_like_this.py .                                                               [ 56%]
tests/test_ordering_common.py .................ss                                            [ 68%]
tests/test_ordering_geo_spatial.py .                                                         [ 68%]
tests/test_pagination.py .                                                                   [ 69%]
tests/test_pip_helpers.py ...                                                                [ 71%]
tests/test_search.py ....ss.....                                                             [ 78%]
tests/test_search_multi_match.py ss....                                                      [ 82%]
tests/test_search_simple_query_string.py ss......ss                                          [ 88%]
tests/test_serializers.py .....                                                              [ 91%]
tests/test_source.py .                                                                       [ 92%]
tests/test_suggesters.py .......                                                             [ 96%]
tests/test_views.py ..                                                                       [ 98%]
tests/test_wrappers.py ...                                                                   [100%]
```

**Comment by CorrosiveKid** [comment]
#149 Should fix the `test_helpers.py` failure.

**Comment by barseghyanartur** [comment]
Solved in 0.20.
